distribution/sdk/ddf-sso-query-widget/src/main/java/org/codice/ddf/ui/Query.java

```diff
@@ -95,21 +95,29 @@ public void setFilterBuilder(FilterBuilder filterBuilder) {
     }
 
     @Override
-    protected void doGet(HttpServletRequest request, HttpServletResponse response)
-            throws ServletException, IOException {
-        doPost(request, response);
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) {
+        try {
+            doPost(request, response);
+        } catch (ServletException | IOException e) {
+            LOGGER.error("Could not post response.");
```

Comment:
>Error messages should have information that can tell a system administrator how to recover from this condition

https://codice.atlassian.net/wiki/spaces/DDF/pages/7045131/Log+Levels#LogLevels-Error

Reply:
Alright, so just put all of these back?